task/buildpacks/0.2/tests/run.yaml

```diff
@@ -1,3 +1,6 @@
+
```

Comment:
Are these changes necessary?

Reply:
there is some issue with the regex, I just did it to run the test